### DISCUSSION TITLE
Indexing recently added only

### DISCUSSION THREAD

**Discussion by slavikator** [photoprism/photoprism#4559] [General]
I don't use the import feature, instead, just copy files from the local machine to the folder with all the photos. After copying, the new files don't show as they aren't indexed yet. So, I go to the library and start indexing. I don't set the checkbox "Full reindex" but it still starts from the earliest photos in library and I need to wait up to two days to see the recently added photos. For some reason, I cannot shose the subfolder where recent photos were added.

Am I missing something?

**Comment by graciousgrey** [comment]
If you index files without the "complete rescan" option only newly added or modified files will be indexed. Others will be skipped. Two days seem to be very long for a normal indexing. 

How big is your library (number of files) and how many new files have you added? Are you using our docker image? What kind of filesystem and hardware?

For general performance tips see:
https://docs.photoprism.app/getting-started/troubleshooting/performance/

**Comment by slavikator** [comment]
You describe expected behavior but it is different in reality. I see the progress and it is indexing folder 2011, then 2012, 2013, and so on till 2024. There are a lot of photos, I mean - a lot. Files recently added around 100.

Photoprism is installed on the Raspberry Pi 4, Raspbian OS (Debian) via apt install, ext4 fs.

**Reply by graciousgrey** [reply]
If the folder you set in the UI as well as disabling the complete rescan are not applied this might be due to wrong permissions for the config files.

Please note that the installations packages are intended for experienced users and maintainers of third-party integrations only, as they [require manual configuration](https://dl.photoprism.app/pkg/linux/README.html#configuration) and [do not include tested system dependencies](https://dl.photoprism.app/pkg/linux/README.html#dependencies). We unfortunately cannot provide support for custom installations and therefore [recommend installing PhotoPrism using docker compose](https://docs.photoprism.app/getting-started/docker-compose/).

**Comment by slavikator** [comment]
I was wrong, it is installed as openmediavault plugin on RaspbianOS.

openmediavault-photoprism 7.0-4
openmediavault PhotoPrism plugin
PhotoPrism® is an AI-powered app for browsing, organizing & sharing your photo collection.
Section: Multimedia
Maintainer: Volker Theile <volker.theile@openmediavault.org>
Homepage: https://www.openmediavault.org/
Repository: openmediavault.org archive/sandworm
Size: 22.43 KiB

**Reply by graciousgrey** [reply]
As mentioned above I would recommend checking the permissions to make sure that PhotoPrism can write settings such as which folder to index or whether to do a normal or complete index to the settings.yml file. 

What exactly happens when you try to select a folder for indexing? Do you get an error? You could also enable [trace logs and watch the logs](https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3) while trying to set a folder for indexing.

If you need help with how to watch logs or check permissions with the openmediavault plugin you can ask in their community forum.

**Reply by slavikator** [reply]
Thank you for your assistance. I will check the permissions. 

The folder selects ok, just cannot select a nested one (to isolate the newest folder for indexing). Only root one can be selected and various nested paths also. But there are no nested folders that I need.
